cmd/kubectl-direct_csi/drives_list.go

```diff
@@ -144,23 +133,15 @@ func listDrives(ctx context.Context, args []string) error {
 	wrappedDriveList := directcsi.DirectCSIDriveList{
 		TypeMeta: metav1.TypeMeta{
 			Kind:       "List",
-			APIVersion: strings.Join([]string{directcsi.Group, directcsi.Version}, "/"),
+			APIVersion: utils.DirectCSIGroupVersion,
 		},
 		Items: filteredDrives,
 	}
-	if yaml {
-		if err := printYAML(wrappedDriveList); err != nil {
+	if printer != nil {
```

Comment:
`printer` will be set to `yaml` mode by default. So, this condition will be `true` always.

Reply:
We can use `yaml || json` condition instead